kaldb/src/main/java/com/slack/kaldb/server/Kaldb.java

```diff
@@ -329,6 +329,7 @@ void shutdown() {
   }
 
   private void addShutdownHook() {
+    LOG.info("Running shutdown hook.");
```

Comment:
This isn't quite accurate - this is adding the shutdown hook, but doesn't invoke it. This will likely be confusing when we're looking at the logs the way it's currently worded.

Reply:
Good catch. Fixed.